tests/en/homeassistant_HassGetState.yaml

```diff
@@ -64,10 +124,22 @@ tests:
         state: "on"
     response: "6"
 
+  - sentences:
+      - "how many lights are on in the kitchen?"
+      - "how many kitchen lights are on?"
+    intent:
+      name: HassGetState
+      slots:
+        area: "Kitchen"
+        domain: "light"
+        state: "on"
+    response: "3"
+
   - sentences:
       - "are all the lights on in the kitchen"
       - "are all lights turned on in the kitchen?"
       - "are all lights in the kitchen on?"
+      - "is the kitchen light on?"
```

Comment:
If we make the suggested changes, this should move to an "any" response and not all.